src/components/Dashboard/NewsItem.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Popup from "reactjs-popup";
+
+const NewsItem = ({ post }) => {
+	const options = { year: 'numeric', month: 'long', day: 'numeric' };
+	return (
+		<tr className="row news-item">
+			<td className="news-item__info">
+				<div className="news-item__info__title">
+					<a href={post.link}>{post.title.rendered}</a>
+				</div>
+				<div className="news-item__info__meta">
+					<span className="news-item__info__meta__modified">{new Date(post.modified).toLocaleDateString( 'en-GB', options )}</span>
+					<span className="news-item__info__meta__byline">by {post._embedded.author[0].name}</span>
```

Comment:
Should be fine but I remember hitting issues where this wasn't set so it's good to wrap a block like this in a conditional eg. `{ post._embedded && post._embedded.author && post._embedded.author.length && <span ...> }`

Or an if block might look cleaner.